BedrockServer.cpp

```diff
@@ -1484,6 +1542,9 @@ void BedrockServer::_control(BedrockCommand& command) {
         suppressCommandPort("SuppressCommandPort", true, true);
     } else if (SIEquals(command.request.methodLine, "ClearCommandPort")) {
         suppressCommandPort("ClearCommandPort", false, true);
+    } else if (SIEquals(command.request.methodLine, "ClearCrashBlacklist")) {
+        unique_lock<decltype(_blackListCommandMutex)> lock(_blackListCommandMutex);
```

Comment:
The command class it a `CrashBlacklist` but the variable is only `_blacklistedCommands`.  Can you rename it to be `_crashBlacklistedCommands` to be consistent?